docs/monthly/2024-01-monthly.md

```diff
@@ -12,150 +12,97 @@ developments to gather their feedback on proposed plans.
 
 ## Roadmap
 
-This month, several items were restructured on the project
-[roadmap](https://github.com/orgs/input-output-hk/projects/21/views/7):
+The Hydra team restructured several items on the project [roadmap](https://github.com/orgs/input-output-hk/projects/21/views/7) this month.
 
 ![The roadmap with features and ideas](./img/2024-01-roadmap.jpg) <small><center>The latest roadmap with features and ideas</center></small>
 
-#### Notable updates
+### Notable updates
```

Comment:
@olgahryniuk @oduameh The smaller font renders better on our docusaurus.

Reply:
Agreed, we followed the heading order rules, but that's not critical at all.